### PR TITLE
Cybereason packaging

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
fixes: #18605
(don't want to link so the issue won't close)

## Description
Just packaging of the Integration 

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Integrations/Cybereason/Cybereason.py [hunk 1]
```diff
@@ -0,0 +1,1478 @@
...
+    elif demisto.command() == 'cybereason-query-file':
+        query_file_command()
+
+    elif demisto.command() == 'cybereason-query-domain':
+        query_domain_command()
+
+    elif demisto.command() == 'cybereason-query-user':
+        query_user_command()
+
+except Exception, e:
```
Comment (yaakovi) [inline Integrations/Cybereason/Cybereason.py]: error handling should be `return_error` and not `raise`.
```suggestion
except Exception as e:
```

**Comment by ronykoz** [comment]
@yaakovi seems strange, I’ll check that on Sunday.
@orlichter1 for the future you can just link the issue using one of the following 
Just copy the url without the fixes prefix 
Copy the url with the relates to prefix
This way the issue will not close once the PR is merged

**Comment by ronykoz** [comment]
It’s because we are using re_create_id_set instead of updating the set, hence not catching the duplication, open an issue to fix this please

**Comment by orlichter1** [comment]
@yaakovi We said we'll fix these issues in the malop-fix branch (which I already did). Please approve here :)

**Pull request merged**